### PR TITLE
 fix(travis): use xvfb service

### PR DESCRIPTION
**Pull request by 9renpoto**
fix: xvfb specs

### CODE REVIEW

**Comment by 9renpoto** [comment]
@cpettitt @lutzroeder hi, cau you review it ? thank you :)

File: .travis.yml [hunk 1]
```diff
@@ -4,4 +4,8 @@ node_js:
 script: KARMA_OPTS="--browsers Firefox,PhantomJS" make -e test
 before_script:
   - export DISPLAY=:99.0
-  - sh -e /etc/init.d/xvfb start
+services:
+  - xvfb
+addons:
+  firefox: latest
+script: KARMA_OPTS="--browsers Firefox" make -e test
```
Comment (9renpoto) [inline .travis.yml:11]: https://travis-ci.com/github/9renpoto/dagre/builds/192685418

**Comment by curran** [comment]
This might close #311

**Review by curran — APPROVED** [review]
LGTM!

**Comment by 9renpoto** [comment]
thanks :)

**Pull request merged**